packages/webview_flutter/webview_flutter_web/README.md

```diff
@@ -0,0 +1,3 @@
+# webview\_flutter\_web
+
+The Web implementation of [`webview_flutter`][1].
```

Comment:
This definitely needs some discussion of the (many) caveats. E.g., explicitly list the supported methods, and say that nothing else works.

It should also a mention of the fact that it's unendorsed, like https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter_web#depend-on-the-package